GNUmakefile

```diff
@@ -25,8 +25,7 @@ fmt:
 
 # Currently required by tf-deploy compile
 fmtcheck:
-	@echo "==> Checking source code against gofmt..."
-	@gofmt -l -s ./$(PKG_NAME)
+	@sh -c "'$(CURDIR)/scripts/gofmtcheck.sh'"
```

Comment:
I thought about this a bit more, and I think you should just do:

```Makefile
fmtcheck: lint
```

As that was probably the intent of the fmt check.

Reply:
@paultyng what do you think?